modules/solid_mechanics/examples/bridge/bridge.e

```diff
@@ -0,0 +1 @@
+../../../tensor_mechanics/examples/bridge/bridge.e
```

Comment:
turns out we can not link the whole directory... Because that directory included a 'tests' file, which ended up getting copied to the new queue directory. Which created a duplicate test the next time the TestHarness was instructed to run.